[BE] User authentication endpoint

MihaiDruta opened the pull request:
[BE] User authentication endpoint
https://trello.com/c/CIYgQHp6/7-be-user-authentication-endpoint

Authentication feature using JWT.

Commit:
[BE] User authentication endpoint
https://trello.com/c/CIYgQHp6/7-be-user-authentication-endpoint

Authentication feature using JWT.

SpartakusMd commented on config/packages/lexik_jwt_authentication.yaml:
```diff
@@ -0,0 +1,5 @@
+lexik_jwt_authentication:
+  secret_key:       '%kernel.project_dir%/config/jwt/private.pem' # required for token creation
+  public_key:       '%kernel.project_dir%/config/jwt/public.pem'  # required for token verification
+  pass_phrase:      'passphrase'
```
Use the paths and passphrase from `.env`. Make sure to copy them to `.env.local`.

Commit:
[BE] User authentication endpoint
https://trello.com/c/CIYgQHp6/7-be-user-authentication-endpoint

Fixed lexik_jwt_authehtication.yaml file.

Commit:
[BE] User authentication endpoint
https://trello.com/c/CIYgQHp6/7-be-user-authentication-endpoint

Fixed lexik_jwt_authehtication.yaml file x2.
